### PR TITLE
Feature/refresh eq surveys

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

**Comment by gemmai95** [comment]
[(3) EQ Survey Refresh](https://trello.com/c/X9JTGGuI/162-3-eq-survey-refresh)

File: console/static/surveys/023.0102.json [hunk 1]
```diff
@@ -13,9 +13,23 @@
...
+    "146b": "In-store / online promotions",
+    "146c": "Special events (e.g. sporting events)",
+    "146d": "Calendar events (e.g. Christmas, Easter, Bank Holiday)",
+    "146e": "Weather",
+    "146f": "Store closures",
+    "146g": "Store openings",
+    "146h": "Other",
     "11": "1/4/2016",
     "12": "31/10/2016",
+    "22": "705000",
```
Comment (warrenbailey) [inline console/static/surveys/023.0102.json]: We shouldn't need to add any questions, the refresh should just be the updated comments ie question 146* etc.

File: console/static/surveys/023.0112.json [hunk 2]
```diff
@@ -8,19 +8,34 @@
...
+    "146h": "Other",
     "51": "84",
     "52": "10",
     "53": "73",
     "54": "24",
     "50": "205",
+    "11": "1/4/2016",
+    "12": "31/10/2016",
+    "22": "705000",
+    "23": "900",
```
Comment (warrenbailey) [inline console/static/surveys/023.0112.json]: again these aren't needed.

File: console/static/surveys/023.0203.json [hunk 3]
```diff
@@ -8,20 +8,29 @@
...
-    "20": "1800000",
-    "21": "60000",
     "22": "705000",
     "23": "900",
     "24": "74",
     "25": "50",
     "26": "100",
-    "146": "some comment"
+    "20": "1800000",
+    "21": "60000"
```
Comment (warrenbailey) [inline console/static/surveys/023.0203.json]: Maybe keep the ordering in its original sequential order

File: console/static/surveys/023.0205.json [hunk 4]
```diff
@@ -8,21 +8,29 @@
...
-    "21": "60000",
     "22": "705000",
     "23": "900",
     "24": "74",
     "25": "50",
     "26": "100",
-    "27": "7000",
-    "146": "some comment"
+    "20": "1800000",
+    "21": "60000",
```
Comment (warrenbailey) [inline console/static/surveys/023.0205.json:25]: As above

**Review by warrenbailey — CHANGES_REQUESTED** [review]
It's great that you've done this as it's really useful to test with. 👍 

However please could you put the 146* questions at the end of the data block so it retains its sequentially ordering and there's a couple of form types that have additional answers which shouldn't be there.

File: console/static/surveys/023.0112.json [hunk 5]
```diff
@@ -8,19 +8,30 @@
...
-    "12": "31/10/2016",
-    "20": "1800000",
     "51": "84",
     "52": "10",
     "53": "73",
     "54": "24",
     "50": "205",
-    "21": "60000"
-  }
+    "11": "1/4/2016",
```
Comment (warrenbailey) [inline console/static/surveys/023.0112.json]: 11 and 12 are still out of order

File: console/static/surveys/023.0205.json [hunk 6]
```diff
@@ -8,21 +8,30 @@
     "instrument_id": "0205",
     "period": "1604"
   },
+  "submitted_at": "2016-03-12T10:39:40Z",
```
Comment (warrenbailey) [inline console/static/surveys/023.0205.json:11]: Do we need to add submitted_at?

**Review by warrenbailey — CHANGES_REQUESTED** [review]
Just a minor one, then it's good to go

File: console/static/surveys/023.0205.json (hunk 6 above)
Comment (gemmai95) [reply console/static/surveys/023.0205.json:11]: The submitted at appears in all the other json example files. Do we not need this?

Comment (warrenbailey) [reply console/static/surveys/023.0205.json:11]: It may not make any difference then.

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**